spec/cb/client_spec.rb

```diff
@@ -51,9 +60,9 @@ module Cb
         end
 
         it 'should call the api using post' do
-          expect(mock_api).to receive(:execute_http_request).
-              with(:post, base_uri, 'parts unknown', {:query=>nil, :headers=>nil, :body=>nil}).
-              and_return(Hash.new)
+          expect(mock_api).to receive(:execute_http_request)
+            .with(:post, base_uri, 'parts unknown', query: nil, headers: nil, body: nil)
+            .and_return({})
```

Comment:
are all these other changes just running rubocop on these file as you modified them?